pubsub/sqs_pub_test.go

```diff
@@ -50,9 +51,10 @@ var _ = Describe("SQS Publisher", func() {
 			done := make(chan interface{})
 			go func() {
 				defer close(done)
-				go testPublisher.Publish(getQueueName(notificationsQueue))
+				go testPublisher.Publish(getQueueName(notificationsQueue), "")
 			}()
 			notificationsCh <- notification
+			time.Sleep(channelWait)
```

Comment:
don't do `Sleep` in tests - the pattern to follow is the one used in the other tests, such as in https://github.com/massenz/go-statemachine/blob/main/pubsub/sqs_pub_test.go#L100-L112